src/components/TracePage/index.js

```diff
@@ -120,6 +120,8 @@ export default class TracePage extends Component {
 
   toggleSlimView(slimView) {
     this.setState({ slimView });
+    // fix issue #12 - TraceView header expander not working correctly
+    setTimeout(() => this.forceUpdate(), 0);
```

Comment:
Agreed this is not ideal, so we'll looking into replacing ReactSticky. Lets just add a // `TODO:` comment here and we can merge it